src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj

```diff
@@ -16,20 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Moq" Version="$(MoqVersion)" />
```

Comment:
You still need `<PackageReference Include="Moq" Version="$(MoqVersion)" />`, sorry if my previous comments around `<Reference` were not clear since I was commenting on the unchanged lines.